paddlenlp/embeddings/token_embedding.py

```diff
@@ -218,19 +224,24 @@ def _extend_vocab(self, extended_vocab_path, vector_np, pad_vector,
     def set_trainable(self, trainable):
         """
         Set the weight of embedding can be trained.
+
         Args:
-            trainable (object: `bool`, required):
+            trainable (`bool`):
                 Whether the weight of embedding can be trained.
```

Comment:
+ The weights can be trained if `trainable` is set to True,  or the weights are fixed if `trainable` is False.

Reply:
done